### PR TITLE
CM-808: Using Keyed Services to Resolve Dependencies in .NET

### PR DESCRIPTION
**Pull request by Banovvv**

### CODE REVIEW

File: dotnet-dependency-injection/UsingKeyedServicesToResolveDependencies/UsingKeyedServicesToResolveDependencies/Program.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+using UsingKeyedServicesToResolveDependencies.Services;
+using UsingKeyedServicesToResolveDependencies.Services.Abstractions;
+
+namespace UsingKeyedServicesToResolveDependencies;
+
+public class Program
```
Comment (fl0tschi) [inline dotnet-dependency-injection/UsingKeyedServicesToResolveDependencies/UsingKeyedServicesToResolveDependencies/Program.cs:6]: We can go for the minimal appearance here as well, so you don't need program class.

File: dotnet-dependency-injection/UsingKeyedServicesToResolveDependencies/UsingKeyedServicesToResolveDependencies/WeatherForecast.cs
Comment (fl0tschi) [inline dotnet-dependency-injection/UsingKeyedServicesToResolveDependencies/UsingKeyedServicesToResolveDependencies/WeatherForecast.cs:1]: I suppose that is a leftover from the initial project. Please remove.

File: dotnet-dependency-injection/UsingKeyedServicesToResolveDependencies/UsingKeyedServicesToResolveDependencies/Properties/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:61594",
+      "sslPort": 0
+    }
+  },
+  "profiles": {
+    "http": {
```
Comment (fl0tschi) [inline dotnet-dependency-injection/UsingKeyedServicesToResolveDependencies/UsingKeyedServicesToResolveDependencies/Properties/launchSettings.json:12]: We should only have HTTPS here. Please remove all the others.

File: dotnet-dependency-injection/UsingKeyedServicesToResolveDependencies/UsingKeyedServicesToResolveDependencies/WeatherForecast.cs
Comment (Banovvv) [reply dotnet-dependency-injection/UsingKeyedServicesToResolveDependencies/UsingKeyedServicesToResolveDependencies/WeatherForecast.cs:1]: Removed.

File: dotnet-dependency-injection/UsingKeyedServicesToResolveDependencies/UsingKeyedServicesToResolveDependencies/Program.cs (hunk 1 above)
Comment (Banovvv) [reply dotnet-dependency-injection/UsingKeyedServicesToResolveDependencies/UsingKeyedServicesToResolveDependencies/Program.cs]: Updated.

File: dotnet-dependency-injection/UsingKeyedServicesToResolveDependencies/UsingKeyedServicesToResolveDependencies/Properties/launchSettings.json (hunk 2 above)
Comment (Banovvv) [reply dotnet-dependency-injection/UsingKeyedServicesToResolveDependencies/UsingKeyedServicesToResolveDependencies/Properties/launchSettings.json]: Sorted.

**Pull request merged**